requirements-dev.txt

```diff
@@ -1,3 +1,5 @@
+boto3==1.34.100
+boto3-stubs==1.34.100
```

Comment:
All of the other stub/type dependencies are grouped together at the bottom of this file.